### PR TITLE
Game

### PR DESCRIPTION
**Pull request by willcurry**

### CODE REVIEW

File: lib/board.rb [hunk 1]
```diff
@@ -1,7 +1,12 @@
 class Board
-  def initialize
-    @board = create_board
+  def initialize(cells = [], last_move = 'o')
+    if cells.empty?
+      @board = create_board
+    else
+      @board = cells
+    end
```
Comment (sarahabimay) [inline lib/board.rb]: You could use a ternary here:
@board = cells.empty? ? create_board : cells

File: lib/board.rb [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
     @turn = 'x'
+    @last_move = last_move
```
Comment (sarahabimay) [inline lib/board.rb]: not sure what 'last_move' indicates having come across it for the first time.. Hopefully all will be revealed, but initially I can't think what it means

Comment (willcurry) [reply lib/board.rb]: I use it to determine the winner

File: lib/board.rb (hunk 1 above)
Comment (willcurry) [reply lib/board.rb]: 👍

**Pull request merged**